services/QuillLMS/app/models/cron.rb

```diff
@@ -17,23 +17,27 @@ def self.interval_1_hour
     StudentLearningSequences::FillMissingWorker.perform_async
   end
 
-  # Configured in Heroku Scheduler to run every day at 07:00UTC
-  # Which is 02:00 or 03:00 Eastern depending on Daylight Savings
   # rubocop:disable Metrics/CyclomaticComplexity
-  def self.interval_1_day
+  def self.run_nondaily_tasks
     run_monday if now.wday == 1
     run_friday if now.wday == 5
     run_saturday if now.wday == 6
     run_weekday if (1..5).include?(now.wday)
     run_school_year_start if now.month == 7 && now.day == 1
     run_monthly if now.day == 1
+  end
+
+  # Configured in Heroku Scheduler to run every day at 07:00UTC
+  # Which is 02:00 or 03:00 Eastern depending on Daylight Savings
+  def self.interval_1_day
+    run_nondaily_tasks
 
     # pass yesterday's date for stats email queries and labels
     date = Time.current.getlocal('-05:00').yesterday
 
     # internal analytics and security
-    DailyStatsEmailJob.perform_async(date)
-    QuillStaffAccountsChangedWorker.perform_async
+    DailyStatsEmailJob.perform_async(date) if production?
```

Comment:
I'm definitely open to which jobs it's better to run on both Prod and Staging. I mainly targetted third-party calls, but I also turned off these internal ones since the staging ones aren't useful. Open to feedback on that.